### DISCUSSION TITLE
guide-R/manipulation/etiquettes-variables

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#35] [Forum]
# guide-R/manipulation/etiquettes-variables

https://larmarange.github.io/guide-R/manipulation/etiquettes-variables.html

**Comment by WilliamZ98** [comment]
Bonjour monsieur, 
Merci pour ces cours.

Je voudrais connaitre les différences entre "subset" et "filter" au delà de la conservation des étiquettes de variables. Et savoir, lequel des deux est le plus pratique d'utiliser.

Respectueusement !

**Reply by larmarange** [reply]
Si vous utilisez dplyr généralement, privilégiez filter()

Sinon utilisez subset() pour éviter de charger dplyr juste pour filter()

Cdlt

**Comment by Azzo35** [comment]
Comment afficher les noms sur le graphe ggplot svp?

**Reply by larmarange** [reply]
Dans la version actuelle de `ggplot2`, le nom des variables (et non l'étiquette de variable) est utilisée pour nommer les axes. Cela peut être modifié manuellement avec `labs()`.

Dans la prochaine version (celle en cours de développement), les étiquettes de variable seront récupérées automatiquement, mais il va falloir patienter un peu.
